.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
+known_numpy=numpy
+known_keras=keras
+known_concurrent=concurrent.futures
+sections=CONCURRENT,FUTURE,STDLIB,KERAS,NUMPY,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+no_lines_before=NUMPY
```

Comment:
Why no lines before numpy?

Reply:
By default the order is for section that are separated with black lines, here I assumed we wanted them in the same section without separation.

Reply:
Oops. I assumed you wanted keras and numpy to be separated.

Reply:
No, I like the default choice: standard libraries first, then third-party libraries, then our libraries.

Reply:
The latest commit uses the default ordering, however the issue with concurrent is still there.